### ISSUE TITLE
Broken dashboards in Kubernetes 1.16

### ISSUE THREAD

**Issue by filip-havlicek** [devopsprodigy/kubegraf#24]
In the Kubernetes 1.16 were removed cadvisor metric labels `pod_name` and `container_name` to match instrumentation guidelines. Instead of these removed labels should  be used labels `pod` and `container` in Prometheus queries.
Kubernetes changelog for 1.16: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md#removed-metrics

All dashboards are affected (except DevOpsProdigy KubeGraf Node's Dashboard) because many graphs uses old metric labels (which doesn't exist in Kubernetes 1.16 anymore).

Old metric labels are also used in dashboard's variables (dashboard settings -> variables). It's case of daemonset's, deployment's and statefulset's dashboards.

## Examples
One example for graph and on for dashboard's variable.
### Graph
in DevOpsProdigy KubeGraf Pod's Dashboard in Memory usage graph:
```
sum (container_memory_usage_bytes{namespace="$namespace", pod_name="$pod", container_name!="", container_name!="POD", container_name=~"$containers"})
```
above query doesn't work in Kubernetes 1.16 and it should be changed to:
```
sum (container_memory_usage_bytes{namespace="$namespace", pod="$pod", container!="", container!="POD", container=~"$containers"})
```
### Variable
in DevOpsProdigy KubeGraf Deployment's Dashboard there is variable `container` defined by this query:
```
label_values(container_memory_usage_bytes{namespace="$namespace", pod_name=~"$deployment-.*", container_name!="", container_name!="POD"}, container_name)
```
for Kubernetes 1.16, it should be:
```
label_values(container_memory_usage_bytes{namespace="$namespace", pod=~"$deployment-.*", container!="", container!="POD"}, container)
```

## Conclusion
As far as I know metrics in Kubernetes 1.15 contains both labels - old with `_name` suffix and new without it.

e. g. container_memory_usage_bytes from Kubernetes 1.15
```
container_memory_usage_bytes{beta_kubernetes_io_arch="amd64",beta_kubernetes_io_os="linux",container="k8s-debug",container_name="k8s-debug",id="redacted",image="redacted",instance="kubetest1n1",job="kubernetes-nodes-cadvisor",kubernetes_io_arch="amd64",kubernetes_io_hostname="kubetest1n1",kubernetes_io_os="linux",name="k8s_k8s-debug_k8s-debug-569c558c57-vzrwc_k8s-debug_51ab9c13-4a3a-499c-a84b-dbec19dfd6ae_0",namespace="k8s-debug",node="kubetest1n1",pod="k8s-debug-569c558c57-vzrwc",pod_name="k8s-debug-569c558c57-vzrwc"}
```

It should be possible easily change label names in Prometheus queries and Dashboards should work for both Kubernetes versions 1.15. and 1.16. I didn't test other versions, but it should also works for newer versions.

**Comment by filip-havlicek** [comment]
Also in Kubernetes 1.14 `pod_name` and `container_name` labels are present alongside `pod` and `container` labels.  Eventual change shouldn't break dashboards for Kubernetes 1.14.

On the other hand in Kubernetes 1.13 and older are only `container_name` and `pod_name` labels.

quick fix in code:
```
sed -i -e "s/container_name/container/g" -e "s/pod_name/pod/g" src/dashboards/* src/components/k8s-page.* dist/dashboards/* dist/components/k8s-page.*
```

**Comment by SergeiSporyshev** [comment]
Hi, @filip-havlicek
Thank you for this issue
We find some solution and we'll add some fixes to the next release

**Comment by SergeiSporyshev** [comment]
Hi @filip-havlicek 
Can you check new version of plugin (https://github.com/devopsprodigy/kubegraf/releases/tag/v1.3.0beta)?

**Comment by filip-havlicek** [comment]
Hi @SergeiSporyshev, thank you.
At first glance, it seems to be fine. I will check it more tomorrow and let you know.

**Comment by SergeiSporyshev** [comment]
Ok, we are waiting your feedback!

**Comment by filip-havlicek** [comment]
I checked it and it looks fine now for both version -  1.15 and 1.16. The target of this issue is fulfilled.

I found another small bug in the Pod's Dashboard in the graph Containers' statuses. There is wrongly defined datasource for this graph which doesn't work because of it (`"datasource": null` instead of `"datasource": "$PromDs"`).
It doesn't relate to this issue. Should I open new one? Or I can send you a pull request, because it's very simple to fix this bug, if you wish.
